### PR TITLE
Create database based on diagram

### PR DESCRIPTION
**Pull request by danielmatama**
We created a database for patients based on the diagram given.
We joined the tables using the "many \-to-many" approach.

### CODE REVIEW

File: schema-based-on-diagram.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  type VARCHAR(255) NOT NULL,
+  name VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE invoices (
+  id SERIAL PRIMARY KEY,
+  total_amount DECIMAL(10, 2) NOT NULL,
+  generated_at TIMESTAMP NOT NULL,
+  medical_history_id INTEGER REFERENCES medical_histories(id)
+);
```
Comment (thecodechaser) [inline schema-based-on-diagram.sql]: - [ ] You missed one entity(column) for this table `payed_at`, Please make sure to add that to complete the table implementation.

image

File: schema-based-on-diagram.sql [hunk 2]
```diff
@@ -0,0 +1,40 @@
+CREATE TABLE medical_histories (
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP NOT NULL,
+  patient_id INTEGER REFERENCES patients(id),
+  status VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
```
Comment (thecodechaser) [inline schema-based-on-diagram.sql:9]: - [x] Please make sure all FOREIGN KEYS have indexes created as requested in the project requirements.

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @danielmatama, and Team👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema-based-on-diagram.sql (hunk 2 above)
Comment (Fene-87) [reply schema-based-on-diagram.sql:9]: Thank you @thecodechaser for your feedback. We have implemented all the changes requested.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**